jdat_notebooks/MRS_Mstar_analysis/JWST_Mstar_dataAnalysis_PointSourceDetectorBasedExtraction.ipynb

```diff
@@ -0,0 +1,2258 @@
+{
```

Comment:
<p>Here is part of the CRDS-related error I saw when I ran this cell off the STScI VPN:</p><p> </p><p><code></code><code></p><p>2022-03-25 08:31:08,510 - CRDS - ERROR -&nbsp;(FATAL) CRDS server connection and cache load FAILED.&nbsp;Cannot continue.</p><p>&nbsp;See https://hst-crds.stsci.edu/docs/cmdline_bestrefs/ or https://jwst-crds.stsci.edu/docs/cmdline_bestrefs/</p><p>&nbsp;for more information on configuring CRDS,&nbsp;particularly CRDS_PATH and CRDS_SERVER_URL. : [Errno 2] No such file or directory: '/grp/crds/cache/config/jwst/server_config'</p><p></code><code></code></p><p>The traceback wound through <code>jwst</code>, <code>asdf</code>, <code>stdatamodels</code>, and other packages. I'm not sure which one caused the root issue.</p><p> </p><p>There may be environment variables that need to be set earlier in the notebook in order for this to work, but I don't know CRDS well enough to have found the solution. If your laptop already has CRDS-related environment variables in <code>~/.bash_profile</code> or a similar file, consider commenting them out to test the experience for a new user.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/dat_pyinthesky/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>